test/Microsoft.AspNetCore.Sockets.Tests/HttpConnectionDispatcherTests.cs

```diff
@@ -150,6 +150,40 @@ public class HttpConnectionDispatcherTests
             }
         }
 
+        [Fact]
```

Comment:
Does this need to be a `ConditionalFact` to disable the test on Win7?

Reply:
No, this doesn't do a real WebSocket connection, it's just setting up a connection context and http context to run through the code path.